backend/keycloak/realm-export.json

```diff
@@ -33,8 +33,8 @@
   "verifyEmail": false,
   "loginWithEmailAllowed": true,
   "duplicateEmailsAllowed": false,
-  "resetPasswordAllowed": false,
-  "editUsernameAllowed": false,
+  "resetPasswordAllowed": true,
+  "editUsernameAllowed": true,
```

Comment:
I am not sure we want to allow them to change their username? I think we just want the username to default to their email address?